bin/binge.js

```diff
@@ -38,7 +38,7 @@ program
 
 program
     .command('cache-clean')
-    .description('cleans the build and install cache')
+    .description('Cleans the build and installs the cache.')
```

Comment:
Here what I wanted to say was really: clean the build cache and clean the install cache. So it doesn't install, it cleans the install cache :)